tasks/install.yml

```diff
@@ -5,12 +5,72 @@
     state: installed
   with_items:
     - java-1.8.0-openjdk
-    - maven
     - ant
     - tomcat
     - tomcat-webapps
     - tomcat-admin-webapps
 
+- name: add JAVA_OPTS to tomcat.conf
+  lineinfile:
+    state: present
+    path: /etc/tomcat/tomcat.conf
+    regexp: '#JAVA_OPTS="-Xminf0.1 -Xmaxf0.3"'
+    line: 'JAVA_OPTS="{{ dspace6_default_java_opt_setting }}"'
+    backup: yes
+    owner: tomcat
+    group: root
```

Comment:
To  follow with the standard that Jason has set on previous roles, we will want to use the wheel group and not the root group. So, owner = tomcat group = wheel. Same applies if the owner is root. owner = root group = wheel.